README.md

```diff
@@ -108,13 +97,13 @@ const optimizely = createInstance({
 
 ## `<OptimizelyProvider>`
 

```

Comment:
out of the scope of your changes, but if you're using the Decide API, you should also link to the latest docs, for which you have to specify v4.0 in the URL (until we make v4.0 the default).
so these links should be updated to v4.0:

so any links with
https://docs.developers.optimizely.com/full-stack/docs/...
should update to
https://docs.developers.optimizely.com/full-stack/v4.0/docs/